user/service.py

```diff
@@ -30,17 +38,18 @@ def create(username: str, password: str) -> Optional[str]:
 
 def update(user_id: str, username: str, password: str):
     """Update a user given its ID."""
-    user = user_dao.get_by_id(user_id)
-
+    user = None
+    try:
+        user = get(user_id)
+    except validation_exceptions.ValidationException as e:
+        raise e
```

Comment:
I think this code behaves exactly the same now as it did before? (except that the traceback would have something like "another exception was raised while handing ...")